filament/src/details/Material.cpp

```diff
@@ -358,6 +368,23 @@ BufferInterfaceBlock::FieldInfo const* FMaterial::reflect(
     return mUniformInterfaceBlock.getFieldInfo(name);
 }
 
+utils::FixedCapacityVector<Variant> FMaterial::getVariants() const noexcept {
+    utils::FixedCapacityVector<Variant> variants;
+
+    filaflat::MaterialChunk const& materialChunk{ mMaterialParser->getMaterialChunk() };
+
+    variants.reserve(materialChunk.getShaderCount());
+    materialChunk.visitShaders([&variants](
+            ShaderModel model, Variant variant, ShaderStage stage) {
+        variants.push_back(variant);
+    });
+    std::sort(variants.begin(), variants.end(),
+            [](Variant lhs, Variant rhs) { return lhs.key < rhs.key; });
+    std::unique(variants.begin(), variants.end());
```

Comment:
Do we need to resize `variants` after calling `std::unique`?

Reply:
uh. yes, I forgot to upload it. thanks!

Reply:
shoot I forgot to remove that code. thanks!